static/css/section/_cloud.scss

```diff
@@ -496,6 +490,51 @@
     }
   }
 
+  // @subsection /cloud/public-cloud
+  &.cloud-public-cloud {
```

Comment:
I can't find any mention of `class="cloud-public-cloud"` in the codebase - do you need to add a whole lot of new styling for it?